scripts/prerelease.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo "Select an option for release："
+echo
+
+select VERSION in Patch Minor Major "Specific Version"
+  do
+    echo
+    if [[ $REPLY =~ ^[1-4]$ ]]; then
+      if [[ $REPLY == 4 ]]; then
+        read -p "Enter a specific version: " -r VERSION
+        echo
+        if [[ -z $REPLY ]]; then
+          VERSION=$REPLY
+        fi
+      fi
+
+      read -p "Release $VERSION - are you sure? (y/n) " -n 1 -r
+      echo
+
+      if [[ $REPLY =~ ^[Yy]$ || -z $REPLY ]]; then
+        # bump version
+        yarn version --new-version $VERSION --no-git-tag-version
+        NEW_VERSION=$(node -p "require('./package.json').version")
+
+        # create new branch
+        git checkout -b $NEW_VERSION master
```

Comment:
I think we should create new branch before bumping version just to be safe.

Reply:
I did this because I wanted to get the new version (after bump) as the branch name. How about naming it to `prerelease` ? We can also delete the existing local `prerelease` branch as well before creating new branch

Reply:
Mm gotcha. Let's leave this as it is. On second thought it's fine because we're not committing yet.